google/cloud/forseti/scanner/scanners/ke_version_scanner.py

```diff
@@ -119,17 +121,27 @@ def _retrieve(self):
             ke_clusters = []
             for ke_cluster in data_access.scanner_iter(
                     session, 'kubernetes_cluster'):
-                service_config = list(data_access.scanner_iter(
-                    session, 'kubernetes_service_config',
-                    parent_type_name=ke_cluster.type_name))[0]
-
                 project_id = ke_cluster.parent.name
                 ke_clusters.append(
                     KeCluster.from_json(project_id,
-                                        service_config.data,
+                                        None,
```

Comment:
Was this field not used before?

Reply:
Correct, it was before.  I moved it into the 2nd query below, in line 143.

Reply:
Got it.